schema/schema.go

```diff
@@ -14,12 +14,17 @@ const (
 	VERSION = "v1"
 )
 
+type Import struct {
```

Comment:
This change in the schema is required in order to support a full translation between the RIDL parser tree and the webrpc AST